src/main/main.ts

```diff
@@ -73,7 +73,7 @@ const createWindow = async () => {
     show: false,
     width: 1024,
     height: 728,
-    icon: getAssetPath('icon.png'),
+    icon: getAssetPath('LockBox_icon.png'),
     webPreferences: {
```

Comment:
Can we instead replace the icon.png file itself? We do not need that anymore right?